packages/manager/modules/manager-components/package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
can you fill README.md file. Add example on how to use the components.

Reply:
Done

Reply:
I don't see where is the documentation ?